include support for recent GCC versions in config and comp scripts
author: Kenneth Hoste (Ghent University)
--- config.orig	2015-03-03 10:33:00.023695000 +0100
+++ config	2015-03-03 10:33:37.858993156 +0100
@@ -452,6 +452,8 @@
             breaksw
          case 4.6:
          case 4.7:
+         case 4.8:
+         case 4.9:
             echo "   Good, the newest gfortran can compile REAL*16 data type."
             breaksw
          default:
@@ -823,6 +825,8 @@
             breaksw
          case 4.6:
          case 4.7:
+         case 4.8:
+         case 4.9:
             echo "   Good, the newest gfortran can compile REAL*16 data type."
             breaksw
          default:
@@ -879,6 +883,8 @@
       breaksw
    case 4.6:
    case 4.7:
+   case 4.8:
+   case 4.9:
       echo "   Good, the newest gfortran can compile REAL*16 data type."
       breaksw
    default:
--- comp.orig	2015-03-03 11:41:57.287089000 +0100
+++ comp	2015-03-03 11:43:02.574753319 +0100
@@ -1612,6 +1612,8 @@
          breaksw
       case 4.6:
       case 4.7:
+      case 4.8:
+      case 4.9:
          set EXTRAOPT="$EXTRAOPT -fno-whole-file"
          breaksw
       default:
@@ -1804,6 +1806,8 @@
             breaksw
          case 4.6:
          case 4.7:
+         case 4.8:
+         case 4.9:
             set EXTRAOPT="$EXTRAOPT -fno-whole-file"
             breaksw
          default:
@@ -2041,6 +2045,8 @@
             breaksw
          case 4.6:
          case 4.7:
+         case 4.8:
+         case 4.9:
             set EXTRAOPT="$EXTRAOPT -fno-whole-file"
             breaksw
          default:
@@ -2099,6 +2105,8 @@
          breaksw
       case 4.6:
       case 4.7:
+      case 4.8:
+      case 4.9:
          set EXTRAOPT="$EXTRAOPT -fno-whole-file"
          breaksw
       default:
